zebra/zebra_routemap.c

```diff
@@ -640,6 +640,42 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(set_kernel_bypass,
+	    set_kernel_bypass_cmd,
+	    "set kernel-bypass",
+	    SET_STR
+	    "kernel bypass\n")
```

Comment:
Very short explanation.. Please imagine yourself as an operator and try to understand what it means.